modules/percolator/src/test/resources/rest-api-spec/test/percolate/15_new.yaml

```diff
@@ -1,46 +0,0 @@
----
-"Basic percolation tests":
-
-  - do:
```

Comment:
What about these? I didn't see them in 5.x but maybe I didn't look super closely.